.gitlab-ci.yml

```diff
@@ -237,26 +237,46 @@ Manifests:
 
 regression-bigiso:
   extends: .regression
+  rules: 
+    # WHITELIST: Run only on x86_64 arch
```

Comment:
If it's only `x86_64` why also specify rhel a centos? I guess you mean `x86_64 and rhel like` systems?